configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,649 @@ benchmark_configs: {
       "}"
   }
 }
+benchmark_configs: {
+  id: "ensure-password-failed-attempts-lockout-includes-root-account"
```

Comment:
Add at least one benchmark doc version (e.g. for generic linux) so our unit tests pass.